validation/standard/standard-config-params-mainnet.toml

```diff
@@ -11,11 +11,6 @@ minimum_base_fee = 1000000000
 system_tx_max_gas = 1000000
 maximum_base_fee = "340_282_366_920_938_463_463_374_607_431_768_211_455"
 
-[l2_output_oracle]
-submission_interval = [1, 86400]
-l2_block_time = [2, 2]
-challenge_period_seconds = [604800, 604800] # i.e. finalization period
-
```

Comment:
We should check which of these params still makes sense to validate (through a different contract) post Fault Proofs, and reinstate those checks.